shared/src/TwinJsonConverter.cs

```diff
@@ -39,7 +39,7 @@ public sealed class TwinJsonConverter : JsonConverter
         private const string CloudToDeviceMessageCountTag = "cloudToDeviceMessageCount";
         private const string AuthenticationTypeTag = "authenticationType";
         private const string X509ThumbprintTag = "x509Thumbprint";
-        private const string ModelId = "digital-twin-model-id";
+        private const string ModelId = "modelId";
 
         /// <summary>
```

Comment:
Will there be separate PR for Device Client?

Reply:
Separate [PR ](https://github.com/Azure/azure-iot-sdk-csharp/pull/1431)because it is for master branch, merged just a minute ago.

Reply:
👍